tests/media_filename_test.go

```diff
@@ -41,6 +41,7 @@ func TestMediaFilenames(t *testing.T) {
 				var filename = filename
 
 				t.Run(fmt.Sprintf("Can download file '%s'", filename), func(t *testing.T) {
+					runtime.SkipIf(t, runtime.Synapse)
```

Comment:
I feel like we would want to keep these tests but change them to use authentication?

I think I'd prefer to disable authenticated media when testing with complement rather than not test these paths?

Reply:
These tests already exist with authentication, see e.g. L56 for this one

Reply:
Ah right, fair enough

Reply:
Can you add a comment or something like "Synapse doesn't support unauthenticated media anymore"?